models/models.js

```diff
@@ -5,3 +5,15 @@ exports.selectCategories = () => {
     return result.rows;
   });
 };
+
+exports.selectReviews = review_id => {
+  if (isNaN(review_id) === true) {
+    return Promise.reject({ status: 400, msg: "invalid review ID" });
```

Comment:
SQL will throw its own error for this situation from db.query (if the data type is not correct for the column), i would lean more towards letting SQL deal with this and catching the error and then acting accordingly

Reply:
this condition check should be unnecessary as Emily mentioned, do we already have a catch block in our controller and error handling for a 400/bad path in our app.js?